elements/lisk-chain/src/constants.ts

```diff
@@ -14,9 +14,9 @@
 
 import { createMessageTag, hash } from '@liskhq/lisk-cryptography';
 
-export const CHAIN_STATE_BURNT_FEE = 'burntFee';
-export const CONSENSUS_STATE_FINALIZED_HEIGHT_KEY = 'finalizedHeight';
-export const CONSENSUS_STATE_VALIDATORS_KEY = 'validators';
+export const CHAIN_STATE_BURNT_FEE = Buffer.from('burntFee', 'utf8');
+export const CONSENSUS_STATE_FINALIZED_HEIGHT_KEY = Buffer.from('finalizedHeight', 'utf8');
+export const CONSENSUS_STATE_VALIDATORS_KEY = Buffer.from('validators', 'utf8');
```

Comment:
@shuse2 To save storage we can also use 1 byte keys here... e.g. `0x01`, `0x02`. Since on storage level every key is just binary buffer so it will save storage and we will still have context in the code.

Reply:
yes, when we apply the state model changes, it will all be bytes. for now, i think this is fine

Reply:
Created new issue https://github.com/LiskHQ/lisk-sdk/issues/6586